docs/src/main/asciidoc/Installation_Guide.adoc

```diff
@@ -0,0 +1,85 @@
+[[Installation_Guide]]
+= Installation Guide
+WildFly team;
+:revnumber: {version}
+:revdate: {localdate}
+:toc: macro
+:toclevels: 3
+:toc-title: Installation Guide
+:doctype: book
+:icons: font
+:source-highlighter: coderay
+:wildflyVersion: 14
+
+:leveloffset: +1
+
+ifndef::ebook-format[:leveloffset: 1]
+
+(C) 2020 The original authors.
+
+ifdef::basebackend-html[toc::[]]
+:numbered:
+
+[[Zipped_Installation]]
+= Installing WildFly from a zipped distribution
+
+That is the historical way to install a complete WildFly server with support for standalone and domain modes. A WildFly distribution
+contains a large number of default configurations allowing you to select the server features and operating modes.
+
+WildFly distribution based installation is well suited when:
+
+* You want to rely on a traditional Jakarta EE application deployment.
+* You want a Jakarta EE compliant server that offers all Jakarta EE features.
+* You are not (yet) concerned by server installation size and memory footprint.
+* You are not yet sure of the kind of operating mode and application you are targeting.
+* Your server instances will contain one or more application deployments.
+
+If that is the kind of installation you are aiming at, the guides that you should read next are:
+
+* link:Getting_Started_Guide{outfilesuffix}[Getting Started Guide] shows you
```

Comment:
I know these changes haven't affected the getting started guide so commenting here.

A common problem users have is "which download to I want?"

It feels a little deep in the docs but I think that is something for later, however the page contains a link to the downloads (that is good) but the listed distributions don't match how they are presented on the download page.

Additionally should the link to the downloads open in a new tab?  If you are reading documentation and click a link to download something the user has likely not finished using the documenation.

Reply:
This point likely needs a Jira for later.  On the getting started page we have a requirements section listing Java 8 or later - we probably need to constrain this a bit better as we are reaching the limits of "later".

We may want a more common section, or more targeted sections.  At some point however I think we will have distributions we say have a finite end point and for true "or later" versions of Java either they must a specific distribution or provision a specific layer set.  i.e. legacy security must be excluded one way or another for true "or later".